docs/reference/search/near-real-time.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[near-real-time]]
+== Near real-time search
```

Comment:
Can we lift some of the content/diagrams from the definitive guide to this page? I think these are difficult concepts for beginners to handle, so we should try and start from the basics and make this page as self-contained as possible. The target audience includes first-time users that just started Elasticsearch, indexed some docs, tried to search for them, and didn't find them.

Reply:
Agreed. I will migrate some content from the NRT page in the definitive guide to help bridge this gap.